src/errors/test-run/index.js

```diff
@@ -158,6 +158,26 @@ export class ActionPositiveIntegerArgumentError extends ActionArgumentErrorBase
     }
 }
 
+export class ActionStringOrStringArrayArgumentError extends ActionArgumentErrorBase {
+    constructor (argumentName, actualValue) {
+        super(TYPE.actionStringOrStringArrayArgumentError, argumentName, actualValue);
+
+        if (Array.isArray(actualValue) && !actualValue.length)
+            this.actualValue = '[]';
+        else if (actualValue === '')
+            this.actualValue = '""';
```

Comment:
`The array argument is expected to be a non-empty string or string array, but it was number.`
Or you worried that `this.actualValue` will become unassigned? It is assigned in `super` constructor.